specs/phase0/fork-choice.md

```diff
@@ -324,6 +349,10 @@ def on_tick(store: Store, time: uint64) -> None:
     store.time = time
 
     current_slot = get_current_slot(store)
+    # Reset store.proposer_score_boost if this is a new slot
+    if store.proposer_score_boost.root != Root():
```

Comment:
It might be more readable to just do

```python
if current_slot > previous_slot:
    store.proposer_score_boost = LatestMessage(root=Root(), epoch=Epoch())
```

I think it's easier to just reason about the fact that the any proposer boost in effect is zeroed out when the slot passes. (rather than check if root is empty and if th current slot does not equal what is in the store

Reply:
done here -- https://github.com/ethereum/consensus-specs/pull/2730/commits/282d85b9e7ea3148c7bae80a11c375d3e7497ecd